src/test/java/org/candlepin/dto/api/v1/ConsumerTranslatorTest.java

```diff
@@ -123,7 +123,7 @@ protected Consumer initSourceObject() {
         consumer.setServiceType("consumer_service_type");
         consumer.setReleaseVer(new Release("releaseVer"));
         consumer.setOwner(owner);
-        consumer.setEnvironment(environment);
+        consumer.addEnvironment(environment);
```

Comment:
We should have multiple environments added here to ensure it translates more than just one.

Reply:
Done.